src/modules/jackrack/filter_vst2.c

```diff
@@ -0,0 +1,268 @@
+/*
+ * filter_ladspa.c -- filter audio through LADSPA plugins
+ * Copyright (C) 2004-2018 Meltytech, LLC
```

Comment:
New files should have "Copyright (C) 2024 Meltytech, LLC"

Reply:
My bad

not only that also the line mention wrong file name `filter_ladspa.c` instead of `filter_vst2.c` this and also the gpl thing is because I copied a lot of ladspa code because both ladspa and vst2 have many similarites especially the plugin discovery code help me a lot.

Hopefully I will address all the reviews in the coming commits.